### DISCUSSION TITLE
Where is API documentation located?

### DISCUSSION THREAD

**Discussion by pablo67340** [ImRidingNow/PavlovRconWebserver#74] [General]
As the title states above, I am looking for the API documentation. I found an issue where someone completed the documentation, but I do not see it linked in the Wiki anywhere. 

I am also having issues installing this on Ubuntu 20.04. I have followed the steps in the Wiki, but apache says service unavailable using the proxy pass built into apache (equivalent). Is somebody able to assist? Can pay for time used.

**Comment by ImRidingNow** [comment]
Hello there is no complete documentation :(
All i can give you documentation wise is:
If you start the service in the Development mode you can go to http://url:port/swagger

Can you follow the steps in the readme and not the wiki?
sry the wiki is outdated and i think its better descripted in the readme xD

- Did you start or made a service for the PavlovRconWebserver?
- Are you on windows or linux?

If you still have problems just write errors or your problems here or in the discord :)
